libs/pika/execution/include/pika/execution/algorithms/drop_operation_state.hpp

```diff
@@ -248,5 +249,19 @@ namespace pika::execution::experimental {
         {
             return detail::partial_algorithm<drop_operation_state_t>{};
         }
-    } drop_operation_state{};
+    };
+
+    /// \brief Releases the operation state of the adaptor before signaling a connected receiver.
+    ///
+    /// Sender adaptor that takes any sender and returns a sender. Values sent as references will be
+    /// copied.
+    ///
+    /// The operation state of previous senders can hold on to allocated memory or values longer
+    /// than necessary which can prevent other algorithms from using those resources.
+    /// @a drop_operation_state can be used to explicitly release the operation state, and thus
```

Comment:
It's not right, but can you make this
```suggestion
    /// @ref drop_operation_state can be used to explicitly release the operation state, and thus
```
for now so it's consistent with other references. I'll look at this separately. I don't think we want this to be in italics.

Reply:
I think removing the anchor would be good. `@ref` should work again after https://github.com/pika-org/pika/pull/1298, but `@p` is also still ok.